conda_forge_tick/update_upstream_versions.py

```diff
@@ -417,3 +418,26 @@ def main(
         n_jobs=n_jobs,
         package=package,
     )
+
+
+def main_find_latest_feedstock_version(
+    name: str = None,
+    node_attrs_file: str = None,
+) -> None:
+
+    if name is not None and node_attrs_file is None:
+        node_attrs_file = f"node_attrs/{name}.json"
+        with LazyJson(node_attrs_file) as node_attrs:
+            attrs = copy.deepcopy(node_attrs.data)
+    elif name is None and node_attrs_file is not None:
+        name = os.path.basename(node_attrs_file)[: -len(".json")]
+        with open(node_attrs_file) as fp:
+            attrs = load(fp)
```

Comment:
This duplicates code from `main_feedstock_parser`. Is there a better way to code this? Also, all my suggestions from above apply here as well.